docs/includes/eloquent-models/PlanetMassAssignment.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use MongoDB\Laravel\Eloquent\Model;
+
+class Planet extends Model
+{
+    /**
+     * The mass assignable attributes.
+     *
+     * @var array
+     */
```

Comment:
@GromNaN quick question:
Should code examples used for documentation always use methods use PHPDoc-style comments? In this case, I thought it wasn't necessary because the user isn't expected to consume the return value in their code.

Reply:
This comment comes from the parent class. It provides the type and description of this property. No strong necessity to keep it.